addons/osfstorage/tests/test_views.py

```diff
@@ -1,203 +1,739 @@
-"""Views tests for the Dropbox addon."""
-import httplib as http
-import unittest
+# encoding: utf-8
+from __future__ import unicode_literals
 
 import mock
+import datetime
+
 import pytest
-from addons.base.tests import views as views_testing
-from addons.dropbox.tests import factories
-from addons.dropbox.tests.utils import (DropboxAddonTestCase, MockDropbox,
-                                        mock_responses, patch_client)
-from dropbox.rest import ErrorResponse
-from framework.auth import Auth
-from nose.tools import assert_equal
-from tests.base import OsfTestCase
-from urllib3.exceptions import MaxRetryError
-from website.addons.dropbox.serializer import DropboxSerializer
-from website.addons.dropbox.views import dropbox_root_folder
+from nose.tools import *  # noqa
+
+from addons.osfstorage.models import OsfStorageFileNode
+from framework.auth.core import Auth
+from website.addons.osfstorage.tests.utils import (
+    StorageTestCase, Delta, AssertDeltas,
+    recursively_create_file,
+)
+from website.addons.osfstorage.tests import factories
+
+from framework.auth import signing
+from website.util import rubeus
+
+from website.models import Tag
+from website.files import models
+from website.addons.osfstorage import utils
+from website.addons.osfstorage import views
+from website.addons.base.views import make_auth
+from website.addons.osfstorage import settings as storage_settings
+
+
+def create_record_with_version(path, node_settings, **kwargs):
+    version = factories.FileVersionFactory(**kwargs)
+    node_settings.get_root().append_file(path)
+    record.versions.append(version)
+    record.save()
+    return record
+
+
+@pytest.mark.django_db
```

Comment:
This is fine, but you can also put 

```python
pytestmark = pytest.mark.django_db
```

At the top of the file to mark all tests in the file.